website/docs/guides/vdc_groups.html.markdown

```diff
@@ -0,0 +1,191 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: VDC Groups"
+sidebar_current: "docs-vcd-guides-vdc-groups"
+description: |-
+  Provides guidance to VDC Group support
+---
+
+# VDC Groups in VMware Cloud Director 10.2+
+
+## About 
+
+Starting with version 10.2, VMware Cloud Director supports data center group networking backed by
+NSX-T Data Center.
+
+To create a network across multiple organization VDCs, you first group the VDCs and then create a
+group network that is shared with them.
+
+Data center group networks backed by NSX-T Data Center provide level-2 network sharing, single
+active egress point configuration, and distributed firewall (DFW) rules that are applied across a
+data center group.
+
+### Data center group 
+
+A data center group acts as a cross-VDC router that provides centralized networking administration,
+egress point configuration, and east-west traffic between all networks within the group. A data
+center group can contain between one and 16 VDCs that you configure to share an active egress point. 
+
+### Availability zone 
+
+An availability zone represents the compute clusters or compute fault domains that are available to
+the network. By default, the availability zone is the provider VDC. 
+
+### Egress point 
+
+An existing NSX-T Data Center Edge Gateway that you configure to connect a data center group to an
+external network. 
+
+## Requirements
+
+VDC Group support requires:
+
+* Terraform Provider VCD 3.6+
+* VMware Cloud Director 10.2+
+
+-> For changed fields (these are usually `vdc` and `owner_id`) the previous behavior is deprecated,
+but still supported. To use VDC Groups though, one needs to migrate to new configuration, which
+shouldn't require rebuilding infrastructure.
+
+## Terraform Provider VCD support 
+
+This document describes features that were introduced in Terraform Provider VCD 3.6.0+ for VDC Group
+support. Earlier versions of Terraform Provider VCD do not support VDC Groups.
+
+Major new approach for VDC Group support is the use of new field `owner_id` (except for routed
+network, which inherits parent VDC/VDC Group from Edge Gateway) field instead of `vdc`. `owner_id`
+field **always takes precedence** above `vdc` field in resource and inherited from `provider`
+section.
+
+### List of resources that support VDC Groups (NSX-T only)
+
+The following list of resources (and their corresponding data sources) support NSX-T VDC Groups (no
+NSX-V VDC Group support is provided):
+
+* [vcd_nsxt_edgegateway](/providers/vmware/vcd/latest/docs/resources/nsxt_edgegateway)
+* [vcd_network_routed_v2](/providers/vmware/vcd/latest/docs/resources/network_routed_v2)
+* [vcd_network_isolated_v2](/providers/vmware/vcd/latest/docs/resources/network_isolated_v2)
+* [vcd_nsxt_network_imported](/providers/vmware/vcd/latest/docs/resources/nsxt_network_imported)
```

Comment:
Do you expect this list to be updated and grow?

CC: @vbauzysvmware @mikeletux